### PR TITLE
Updated Dewan's os guide (prod)

### PR DESCRIPTION
**Pull request by jakub-pomykala**

### CODE REVIEW

File: instructions/cloud-hosted-guide-rest-openshift/instructions.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Building and deploying a RESTful web service on OpenShift 4.x
 
+When you create a new REST application, the design of the API is important. The JAX-RS APIs can be used to create JSON-RPC, or XML-RPC APIs, but it wouldn't be a RESTful service. A good RESTful service is designed around the resources that are exposed, and on how to create, read, update, and delete the resources. The service responds to **GET** requests to the **/LibertyProject/System/properties** path. The **GET** request should return a **200 OK** response that contains all of the JVM's system properties.
```
Comment (austin0) [inline instructions/cloud-hosted-guide-rest-openshift/instructions.md]:
```suggestion
When you create a new REST application, the design of the API is important. A good RESTful service is designed around the resources that are exposed, and on how to create, read, update, and delete the resources. The service responds to **GET** requests to the **/LibertyProject/System/properties** path. The **GET** request should return a **200 OK** response that contains all of the JVM's system properties.
```

File: instructions/cloud-hosted-guide-rest-openshift/instructions.md [hunk 2]
```diff
@@ -95,26 +97,63 @@ The configuration does the following actions:
...
+Add the above to the end of the **properties** section (Add a new line at the end of **line 21** and paste the above code in).
+
+Next update the **server** config file.
+
+> From the menu of the IDE, select
+  **File** > **Open** > guide-docker/finish/src/main/liberty/config/server.xml
+
+```
+  <!-- tag::webApplication[] -->
+  <webApplication location="guide-rest-intro.war" contextRoot="/LibertyProject"/>
```
Comment (austin0) [inline instructions/cloud-hosted-guide-rest-openshift/instructions.md]:
```suggestion
  <webApplication location="guide-docker.war" contextRoot="/LibertyProject"/>
```

Should this be guide-docker?

File: instructions/cloud-hosted-guide-rest-openshift/instructions.md [hunk 3]
```diff
@@ -243,24 +299,33 @@ oc get imagestreams
...
+## Nice Work!
+
 In this lab, you learned:
 
 1. How to build a Java/OpenLiberty REST service with JAX-RS and JSON-B
 2. How to build and run the app locally
 3. How to build and deploy the app on Red Hat OpenShift 4.X
 
+Nice work! You have learned how to develop a REST service in Open Liberty by using JAX-RS and JSON-B and then deployed the application on OpenShift 4.x.
+
```
Comment (austin0) [inline instructions/cloud-hosted-guide-rest-openshift/instructions.md]: Same content repeated twice

**Review by austin0 — CHANGES_REQUESTED** [review]

File: instructions/cloud-hosted-guide-rest-openshift/instructions.md (hunk 3 above)
Comment (jakub-pomykala) [reply instructions/cloud-hosted-guide-rest-openshift/instructions.md]: done

File: instructions/cloud-hosted-guide-rest-openshift/instructions.md (hunk 2 above)
Comment (jakub-pomykala) [reply instructions/cloud-hosted-guide-rest-openshift/instructions.md]: actually it should be `rest.war`

**Comment by jamiecoleman92** [comment]
Happy to merge so I am able to test on the competition site.

**Pull request merged**